components/ai_chat/resources/page/components/filter_menu/tools_menu.test.tsx

```diff
@@ -60,41 +89,164 @@ describe('tools_menu', () => {
     query: null,
   }
 
-  it('should render', () => {
-    const { getByText, container } = render(<ToolsMenu {...defaultProps} />)
-    expect(getByText('category 1')).toBeInTheDocument()
-    expect(getByText('category 2')).toBeInTheDocument()
-    expect(getByText('subheading 1')).toBeInTheDocument()
-    expect(getByText('subheading 2')).toBeInTheDocument()
-    expect(getByText('entry 1')).toBeInTheDocument()
-    expect(getByText('entry 2')).toBeInTheDocument()
-    expect(getByText('entry 3')).toBeInTheDocument()
-
-    expect(container.querySelector('leo-buttonmenu')).toHaveProperty(
-      'isOpen',
-      true,
+  const renderWithContext = (
```

Comment:
hey so this shouldn't be using the context (see https://github.com/brave/brave-core/pull/31416#discussion_r2389442088)